vision/object_detection_segmentation/yolov2-coco/README.md

```diff
@@ -0,0 +1,32 @@
+# YOLOv2-COCO
```

Comment:
Please also include a link to this folder from the README in the source of this repository and describe the difference between this offering and the other YOLOv2 model link in the description.

Reply:
Thank you for your comment. I revised README to add a link to the yolov2-coco folder.